banzai/image/header.py

```diff
@@ -0,0 +1,25 @@
+from banzai import logs
+from banzai.utils import date_utils, fits_utils
+
+logger = logs.get_logger(__name__)
+
+class Header(object):
+    def __init__(self, header: dict):
+        self.request_number = header['REQNUM']
```

Comment:
I'm assuming that these values will always be in the header. If not, a `KeyError` will be raised which can be wrapped with something more descriptive.

If these values will not always be in the header, they should have defaults like at the end of this constructor.